### DISCUSSION TITLE
best practices for github actions with Symfony bundles

### DISCUSSION THREAD

**Discussion by tacman** [symfony/symfony#53690] [Q&A] [answered]
I'm trying to setup github actions for my bundles, and am shamelessly copying @kbond 's https://github.com/zenstruck/.github.  

All the zenstruck bundles have a ci.yml file very similar to https://github.com/zenstruck/schedule-bundle/blob/1.x/.github/workflows/ci.yml

I've tried to mimic this behavior, but my actions fail:
    
Error: Unable to resolve action `survos/.github@php-test-symfony`, unable to find version `php-test-symfony`

https://github.com/survos/BarcodeBundle/actions/runs/7719078257/job/21041649442

It should be pointing to https://github.com/survos/.github/blob/main/.github/workflows/php-test-symfony.yml

With Symfony 6.3 becoming EOL tomorrow, I'm trying to get all my bundles to have some semblance of CI and make sure they work (and continue to work) with Symfony 6.4 and 7.  Any suggestions as to what I'm doing wrong?

**Comment by mbabker** [comment, marked as answer]
`zenstruck/.github@php-test-symfony` is referring to a branch on that repo it looks like, specifically https://github.com/zenstruck/.github/tree/php-test-symfony

**Reply by tacman** [reply]
Thanks!  I thought that was the name of the job.

**Comment by kbond** [comment]
I believe there is a way to avoid the need for branches (and use sub-directories) but I haven't experimented with that yet.
